### PR TITLE
feature/test2

### PR DESCRIPTION
**Pull request by Kgomotso196**
# PROJECT TO DOs
### IN THIS PROJECT WE:

1. Have written unit tests for a JavaScript app.
2. Have tested functions responsible for adding and deleting list items.
    - By creating a test file (functionality.test.js) for a file containing the add item and delete item functions that you want to test.
3. Tested functions responsible for the status and content updates.
4. Tested the order re-position of items in the list upon drag/drop actions.
5. Mocked the HTML elements to test DOM manipulation functions.
6. Made sure we group our tests using the describe() method.
7. Made sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
8. Made sure that we documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*eslint-disable*/
```
Comment (LionRouge1) [inline src/index.js]: - [ ] Implementing linters in your project will help ensure that the quality of your written code is good. Therefore, I suggest you don't disable linter checking, but rather find a way to improve your code.

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 import _ from 'lodash'; /*eslint-disable*/
```
Comment (LionRouge1) [inline src/index.js]: - [ ] Same here too as pointed out earlier :point_up:

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @Kgomotso196, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good job writing test for the add and delete functionalities :1st_place_medal:  
- Nice work testing all the pure function in the project :+1: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @Kgomotso196 @cosywasswa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Made Requested Changes :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive Pr :heavy_check_mark: 
Good Commit Messages :heavy_check_mark: 

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**